google/cloud/spanner_dbapi/cursor.py

```diff
@@ -298,6 +321,11 @@ def fetchone(self):
             return None
 
     def fetchall(self):
+        """Fetch all elements.
+
+        :rtype: list
+        :returns: A list of fetched elements.
```

Comment:
Same here, use the [definition](https://www.python.org/dev/peps/pep-0249/#fetchall) to better phrase it.